ucSubmitNews.ascx.vb

```diff
@@ -336,6 +342,38 @@ Namespace Ventrian.NewsArticles
 
         End Sub
 
+        Private Sub BindTags()
+            If (ArticleSettings.UseStaticTagsListSubmit) Then
+                txtTags.Visible = False
+                lblTags.Visible = False
+
+                Dim objTagController As New TagController
+                Dim objTags As ArrayList = objTagController.List(ModuleId, Null.NullInteger)
+
+                objTags.Sort()
+                lstTags.DataSource = objTags
+                lstTags.DataBind()
+            Else
+                lstTags.Visible = False
+            End If
+        End Sub
+
+        Private Sub SelectTags(ByVal tagList As String)
+            Dim objTagController As New TagController
+            For Each tag As String In tagList.Split(","c)

```

Comment:
Do we need to check for the case when `tagList` is `Nothing`, or does it always come across as an empty string if it's not set?

Also, consider using [the `Split` overload that takes a `StringSplitOptions`](https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=net-5.0#System_String_Split_System_Char___System_StringSplitOptions_) to avoid having to handle empty tags.

Reply:
Yes, `ArticleInfo.Tags` is sourced from `dbo.Ventrian_NewsArticles_SplitTags(@ArticleID int)` which will always default to an empty string.

And good call 👍